sdks/java/io/solace/src/main/java/org/apache/beam/sdk/io/solace/broker/SempBasicAuthClientExecutor.java

```diff
@@ -52,7 +56,7 @@
  * response is 401 Unauthorized, the client will execute an additional request with Basic Auth
  * header to refresh the token.
  */
-class SempBasicAuthClientExecutor implements Serializable {
+public class SempBasicAuthClientExecutor implements Serializable {
```

Comment:
This class doesn't need to be serialized, right?

Reply:
Can we leave it out for a separate PR?

Reply:
That's fine if this class isn't part of the public API, otherwise I'd fix it right now before users somehow introduce a dependency on the premise of serializability of this class.

Reply:
Removed, done. I also removed "serializable" from the SempClient class https://github.com/apache/beam/pull/32400/files#diff-720dd30a84a8f020c23124a591126f82ba7a4484fe1130d34029882b5e3373ebL29

Reply:
Edit, the tests are failing now. I think it has to be serializable, as this is a field in SempClient, which has to be serializable. I'll revert the changes

Reply:
Ack, LGTM then.